File: src/model/TransportUnit.java
package model;

public class TransportUnit {
    
    private long id;
    private double unitType;
    private long customerId;
    
    public TransportUnit(long id, long customerId, double unitType) {
        this.id = id;
        this.unitType = unitType;
        this.customerId = customerId;
    }

    /**
     * @return the id
     */
    public long getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(long id) {
        this.id = id;
    }

    /**
     * @return the unitType
     */
    public double getUnitType() {
        return unitType;
    }
    

}

File: Model/TransportUnit.cs
using System;
using System.Runtime.Serialization;

namespace Model
{
    [DataContract()]
    public class TransportUnit
    {

        [DataMember()]
        public long ID { get; set; }
        [DataMember()]
        public long CustomerID { get; private set; }
        [DataMember()]
        public double UnitType { get; private set; }
        
        public TransportUnit(long id, long customerID, double unitType) {
            this.ID = id;
            this.UnitType = unitType;
            this.CustomerID = customerID;
        }

        public TransportUnit(long customerID, double unitTyoe)
        {
            this.CustomerID = customerID;
            this.UnitType = unitTyoe;
        }
    }
}

